doc/array/sort.md

```diff
@@ -1,2 +1,36 @@
 @property {function} can-observe/sort sort
 @parent can-observe/array
+
+@description Add elements to the end of an observe array.
+@signature `list.sort([compareFunction])`
+
+  `sort` sorts the elements in place and returns the newly sorted array. The API is the same as the native JavaScript `Array.prototype.sort` API.
```

Comment:
I would remove “newly” because it almost makes it sound like it’s a new sorted array.

Maybe link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort?